### PR TITLE
Updating the JSON to unify format of variables

### LINKED ISSUE #4464: Unify the word cases in our JSON files
the oldest JSON files use camelCase but the latest style is to use PascalCase to match the C# source code, so this'd be a nice small housekeeping task to update all the old JSON files to use the new name cases everywhere.

### PR DESCRIPTION
**Pull request by elj40**
**Updating the JSON to unify format of variables**

Converts all the second layer variables into PascalCase instead of camelCase
Ignore first layer and any variables of length 1 (e.g. x, y, z)

**Related Issues**

closes #4464 

Unsure if updating keys in compounds.json will affect the selection in bio_processes.json

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

File: simulation_parameters/microbe_stage/organelles.json [hunk 1]
```diff
@@ -1,279 +1,279 @@
...
+    "ProkaryoteChance": 0.5,
+    "ChanceToCreate": 0.5,
+    "Mass": 0.3,
+    "DisplayScene": "res://assets/models/organelles/Pilus.tscn",
+    "Name": "ORGANELLE_PILUS",
+    "IconPath": "res://assets/textures/gui/bevel/parts/PilusIcon.png",
+    "EditorButtonGroup": "External",
+    "EditorButtonOrder": 2,
+    "AvailableUpgrades": {
+      "None": {
```
Comment (hhyyrylainen) [inline simulation_parameters/microbe_stage/organelles.json:28]: Upgrade names are also internal names and will break things if they are capitalized.

Comment (elj40) [reply simulation_parameters/microbe_stage/organelles.json:28]: So all names inside the `AvailableUpgrades` object should return to camelCase?

Comment (hhyyrylainen) [reply simulation_parameters/microbe_stage/organelles.json:28]: Yes, the keys directly inside the upgrades dictionary are internal keys referred to by exact values in the C# code.

**Review by elj40 — COMMENTED** [review]
Took all variables inside `AllUpgrades` back to camelCase

**Pull request closed without merging**